wagtailvideos/models.py

```diff
@@ -31,11 +31,39 @@
 logger = logging.getLogger(__name__)
 
 
+class VideoQuality(ChoiceEnum):
+    default = 'Default'
+    lowest = 'Low'
+    highest = 'High'
+
+
 class MediaFormats(ChoiceEnum):
     webm = 'VP8 and Vorbis in WebM'
     mp4 = 'H.264 and MP3 in Mp4'
     ogg = 'Theora and Voris in Ogg'
 
+    def get_quality_param(self, quality):
+        if self is MediaFormats.webm:
+            if quality is VideoQuality.lowest:
```

Comment:
Enums can be used as dict keys, so you could do something like:

```
return {
    VideoQuality.lowest: '50',
    VideoQuality.default: '22',
    VideoQuality.highest: '4',
}[quality]
```

although now that I've typed it out, it is actually no shorter at all...